arch/arm/src/stm32h5/Kconfig

```diff
@@ -625,6 +633,115 @@ config STM32H5_USBFS
 
 endmenu # STM32H5 Peripheral Selection
 
+menu "DTS Configuration"
+	depends on STM32H5_DTS
```

Comment:
change to
```
if STM32H5_DTS
menu "DTS Configuration"
...
endif
```
and remove ALL `depends on STM32H5_DTS`.

Reply:
since every option add the dependence on STM32H5_DTS, one pair of if/endif could guard all related options